### ISSUE TITLE
Apothecary's Gloves new glove base not showing in PoB when imported

### ISSUE THREAD

**Issue by staylifted123** [PathOfBuildingCommunity/PathOfBuilding#2627]
* ** Imported multiple builds using the Apothecary's Gloves and it shows as nothing equipped in the gloves section
* ** Import a build from both pathofexile.com and poe.ninja that included Apothecary's Gloves
* ** Import a build from both pathofexile.com or poe.ninja that includes Apothecary's Gloves
* ** Yes
* ** Version 2.2.1
* ** https://imgur.com/a/enXXCPZ

**Comment by CJCombrink** [comment]
Related #2612

A copy of the item from the game:
```
Item Class: Gloves
Rarity: Rare
Corpse Hold
Apothecary's Gloves
--------
Quality: +10% (augmented)
Armour: 190 (augmented)
Energy Shield: 48 (augmented)
--------
Requirements:
Level: 70
Str: 46
Int: 58
--------
Sockets: G-B-R-R 
--------
Item Level: 73
--------
17% increased Damage over Time (implicit)
--------
+93 to Accuracy Rating
66% increased Armour and Energy Shield
+41% to Lightning Resistance
+30% to Chaos Resistance
```

When trying to create it the Create button is disabled and states The item is invalid

**Comment by Wires77** [comment]
Fixed in 2.3.0
